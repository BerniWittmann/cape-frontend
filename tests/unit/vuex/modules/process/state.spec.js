import processState from '@/vuex/modules/process/state'

describe('Vuex', () => {
  describe('Modules', () => {
    describe('Process', () => {
      describe('State', () => {
        it('has an empty base state', () => {
          expect(processState).toMatchSnapshot()
        })
      })
    })
  })
})
