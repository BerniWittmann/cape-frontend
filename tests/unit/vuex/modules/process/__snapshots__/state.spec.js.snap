// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vuex Modules Process State has an empty base state 1`] = `
Object {
  "activeProcess": undefined,
  "processes": Array [],
}
`;
